src/pages/404.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Heading, Stack, VStack, Text } from '@chakra-ui/react'
+import { NextSeo } from 'next-seo'
+import Image from 'next/image'
+import NextLink from 'next/link'
+
+import mascotLogo from '@app-public/mascot-logo.png'
+import { CTAButton } from '~/components/CTAButton'
+import FluidContainer from '~/components/FluidContainer'
+import Layout from '~/components/Layout'
+import Link from '~/components/Link'
+
+const Custom404 = () => (
+  <Layout>
+    <NextSeo title="404 - Not Found" />
+    <Box pb={{ base: 8, lg: 16 }} align="center">
+      <FluidContainer>
+        <VStack px="10">
+          <Image
+            src={mascotLogo}
```

Comment:
On Figma I used a placeholder image without the "Octoclairvoyant" text baked in. I think we can rely on the `h1` in the nav-bar to make clear that people are on the "Octoclairvoyant website".

The current image is too small/low resolution to use, but maybe you can make a new logo without the "Octoclairvoyant" text baked in?

Here's the image I used on Figma:

https://github.com/octoclairvoyant/octoclairvoyant-webapp/blob/main/public/mascot-icon.png

Reply:
True, didn't realize about the image without text. Indeed, the current image icon is too small, but I have the original one and I can replace it to get a bigger one.